### PR TITLE
A couple of notes to help beginning developers get started

### PR DESCRIPTION
**Pull request by dsblank**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ jupyter lab build  # Build the app dir assets (optional)
 
 Notes:
 
+* A few of the scripts will run "python". If your target python is called something else (such as "python3") then parts of the build will fail. You may wish to build in a conda environment, or make an alias.
```
[the remark this answers is not in the document]
Comment (dsblank) [reply CONTRIBUTING.md]: Sure; things I had to change:

```
./docs/source/conf.py:#!/usr/bin/env python3
./setupbase.py:#!/usr/bin/env python3
./setup.py:#!/usr/bin/env python3
./buildutils/lib/ensure-repo.js:    let cmd = 'python3 setup.py --version';
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -82,6 +82,10 @@ jupyter lab build  # Build the app dir assets (optional)
...
+
+* There are versions of Node that are too modern. You should use a version of Node < 10.
```
[the remark this answers is not in the document]
Comment (dsblank) [reply CONTRIBUTING.md]: Yes, as I am new to juyterlab, I am not sure what is my confusion, a bug, or just the way it has to be. These are just notes to help me and others figure it out.

**Comment by jzf2101** [comment]
We're working on node 10 in https://github.com/jupyterlab/jupyterlab/issues/4585

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jzf2101) [reply CONTRIBUTING.md]: Thanks.  What error msg do you get with that?

**Pull request merged**